index.html

```diff
@@ -21,8 +21,9 @@ <h1><span class="text-muted">Feedback Machine</span></h1>
           <button class="btn btn-default" onclick="location.href='connect_four.html'">Connect Four</button>
           <button class="btn btn-default" onclick="location.href='black_thursday.html'">Black Thursday</button>
           <button class="btn btn-default" onclick="location.href='night_writer.html'">Night Writer</button>
+          <button class="btn btn-default" onclick="location.href='mid_mod.html'">Mid-Mod</button>
```

Comment:
I wonder if we could use the same page for both paired assessments..? I think the rubric is the same anyway.